modules/alloy/src/main/ts/ephox/alloy/alien/ComponentStructure.ts

```diff
@@ -2,11 +2,14 @@ import { Fun } from '@ephox/katamari';
 import { Compare, PredicateExists, SugarElement } from '@ephox/sugar';
 
 import { AlloyComponent } from '../api/component/ComponentApi';
+import * as AriaControls from '../aria/AriaControls';
 import * as AriaOwner from '../aria/AriaOwner';
 import { AnchorSpec } from '../positioning/mode/Anchoring';
 
 const isAriaPartOf = (component: AlloyComponent, queryElem: SugarElement<Node>): boolean =>
-  AriaOwner.find(queryElem).exists((owner) => isPartOf(component, owner));
+  AriaOwner.find(queryElem)
```

Comment:
What drove the decision to use either one here? Are we using aria-owner somewhere else? Or just a precaution?

Reply:
Good question... I didn't look into that. I'll see where else it's used, as it's possible we don't need the `AriaOwner` module at all anymore.

Reply:
So looks like it's used by the `FloatingToolbarButton` schema, for which `aria-owner` would be incorrect there again so we should swap that to use `aria-controls` as well. I'll swap it out and just drop all this back to a single `AriaControls` module.